ZCS-1417

mihaipopahtp opened the pull request:
- adding Vietnamese translation
- setting correct encoding for com_zimbra_about

silentsakky commented on src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder_vi.properties:
```diff
@@ -0,0 +1,79 @@
+# Preferences dialog
+#
+BirthdayReminder_zimletPreferences = Tr\u00ecnh nh\u1eafc sinh nh\u1eadt: T\u00f9y ch\u1ecdn
+BirthdayReminder_PleaseNote = L\u01b0u \u00fd: Zimlet n\u00e0y s\u1ebd t\u1ef1 \u0111\u1ed9ng kh\u1edfi t\u1ea1o m\u1ed9t l\u1ecbch g\u1ecdi l\u00e0 <b>'{0}'</b> <br/> \u0111\u1ec3 l\u01b0u m\u1ecdi l\u1eddi nh\u1eafc nh\u1edf sinh nh\u1eadt.
```
do we need double single quotes here? as mentioned in other pull requests

mihaipopahtp replied:
English source doesn't have them either and it's the same for all other languages.
Should I fix them in all files + English? Is there a ticket for this?

silentsakky replied:
not sure let's discuss this offline

silentsakky commented on src/zimlet/zimbra_inboxzero/zimbra_inboxzero_vi.properties:
```diff
@@ -0,0 +1,60 @@
+# NOTE: locale. These messages are here mainly just to keep the
+# NOTE: supporting code clean of hard-coded strings.
+
+archiveFolder = \u0110\u00e3 l\u01b0u tr\u1eef
```
above warning says we should not translate these strings

tarangkhandelwal commented on src/zimlet/com_zimbra_about/com_zimbra_about.properties:
```diff
@@ -16,5 +16,5 @@
 label = About Zimbra
 description = Displays Zimbra version
-copyright = Copyright � 2005-2011 VMware, Inc. VMware and Zimbra are registered trademarks or trademarks of VMware, Inc. 
+copyright = Copyright © 2005-2011 VMware, Inc. VMware and Zimbra are registered trademarks or trademarks of VMware, Inc. 
```
this shouldn't be having VMware I guess. @silentsakky ?

silentsakky replied:
I guess we are not using this zimlet anymore

mihaipopahtp replied:
I only fixed the encoding here because it messed up my script. It was the only file with wrong encoding. 
It's not translated in any language, so I don't think it matters. Changed it to Synacor, just in case.

silentsakky replied:
thx

silentsakky commented on src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder.properties:
```diff
@@ -38,7 +38,7 @@ BirthdayReminder_formatIncorrect = Birthdays must have "YYYY-MM-DD" format
 #
 BirthdayReminder_zimletPreferences = Birthday Reminder: Preferences
-BirthdayReminder_PleaseNote = Note: This Zimlet automatically creates a Calendar called <b>'{0}'</b> <br/> to store all birthday reminders.
+BirthdayReminder_PleaseNote = Note: This Zimlet automatically creates a Calendar called <b>''{0}''</b> <br/> to store all birthday reminders.
```
should we use smart quotes?

mihaipopahtp replied:
will be fixed with ZCS-2592 in all files

silentsakky replied:
thanks

silentsakky commented on src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder_ar.properties:
```diff
@@ -38,7 +38,7 @@ BirthdayReminder_formatIncorrect = \u064a\u062c\u0628 \u0623\u0646 \u062a\u0643\
 #
 BirthdayReminder_zimletPreferences = \u062a\u0630\u0643\u064a\u0631 \u064a\u0648\u0645 \u0627\u0644\u0645\u064a\u0644\u0627\u062f:\u200f \u0627\u0644\u062a\u0641\u0636\u064a\u0644\u0627\u062a
-BirthdayReminder_PleaseNote = \u0645\u0644\u0627\u062d\u0638\u0629:\u200f \u062a\u0639\u0645\u0644 \u0628\u0631\u0645\u062c\u064a\u0629 Zimlet \u0647\u0630\u0647 \u0639\u0644\u0649 \u0625\u0646\u0634\u0627\u0621 \u062a\u0642\u0648\u064a\u0645 \u062a\u0644\u0642\u0627\u0626\u064a\u064b\u0627\u060c \u064a\u0633\u0645\u0649 <b>'{0}'</b> <br/> \u0644\u062a\u062e\u0632\u064a\u0646 \u0643\u0627\u0641\u ...
+BirthdayReminder_PleaseNote = \u0645\u0644\u0627\u062d\u0638\u0629:\u200f \u062a\u0639\u0645\u0644 \u0628\u0631\u0645\u062c\u064a\u0629 Zimlet \u0647\u0630\u0647 \u0639\u0644\u0649 \u0625\u0646\u0634\u0627\u0621 \u062a\u0642\u0648\u064a\u0645 \u062a\u0644\u0642\u0627\u0626\u064a\u064b\u0627\u060c \u064a\u0633\u0645\u0649 <b>''{0}''</b> <br/> \u0644\u062a\u062e\u0632\u064a\u0646 \u0643\u0627\u0641 ...
```
same as above

silentsakky commented on src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder_ca.properties:
```diff
@@ -38,7 +38,7 @@ BirthdayReminder_formatIncorrect = Els aniversaris han de tenir el format "AAAA-
 #
 BirthdayReminder_zimletPreferences = Recordatori d\u2019aniversari: Prefer\u00e8ncies
-BirthdayReminder_PleaseNote = Nota: Aquest Zimlet crea autom\u00e0ticament una agenda anomenada <b>{0}</b> <br/> per desar-hi tots els recordatoris d\u2019aniversaris.
+BirthdayReminder_PleaseNote = Nota: Aquest Zimlet crea autom\u00e0ticament una agenda anomenada <b>''{0}''</b> <br/> per desar-hi tots els recordatoris d\u2019aniversaris.
```
same as above

silentsakky committed:
ZCS-1417
- adding Vietnamese translation
- setting correct encoding for com_zimbra_about
- string fixes
- squashing ZCS-2592 updates to smart quotes + minor fixes

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Zimbra/zm-zimlets?pullRequest=6) <br/>All committers have signed the CLA.
